shared/postgres/compose.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  benchi-postgres:
+    container_name: benchi-postgres
+    networks:
+      - benchi
+
+    image: debezium/postgres:16
```

Comment:
It looks like the image was moved from Docker Hub to Quay: https://hub.docker.com/r/debezium/postgres

Reply:
Thank you. I had missed that.